tests/mock_module/permissions.py

```diff
@@ -23,6 +25,9 @@ class PermissionPolicy(RecordPermissionPolicy):
     can_update = [AnyUser(), SystemProcess()]
     can_delete = [AnyUser(), SystemProcess()]
     can_create_files = [AnyUser(), SystemProcess()]
+    can_set_content_files = [AnyUserIfFileIsLocal(), SystemProcess()]
+    can_get_content_files = [AnyUserIfFileIsLocal(), SystemProcess()]
+    can_commit_files = [AnyUserIfFileIsLocal(), SystemProcess()]
     can_read_files = [AnyUser(), SystemProcess()]
     can_update_files = [AnyUser(), SystemProcess()]
-    can_delete_files = [AnyUser(), SystemProcess()]
+    can_delete_files = [AnyUserIfFileIsLocal(), SystemProcess()]
```

Comment:
minor: this might not be needed actually, since that could be a way to "cancel" the background upload?